can/interfaces/socketcand/socketcand.py

```diff
@@ -231,3 +318,11 @@ def shutdown(self):
         """Stops all active periodic tasks and closes the socket."""
         super().shutdown()
         self.__socket.close()
+
+    @staticmethod
+    def _detect_available_configs() -> List[can.typechecking.AutoDetectedConfig]:
+        try:
+            return detect_beacon()
```

Comment:
I'm not sure, whether it's a good idea to delay the `can.detect_available_configs()` by 12 seconds for everyone.
Is it possible to reduce that duration?

Reply:
Ok, I understand what you're saying. I wasn't aware calling can.detect_available_configs() will call all the detect_available_configs of all the interfaces. hehe. Yeah - we could reduce it. Right now the time window is designed to be wide enough for 4 beacon broadcasts. If we reduce that to 2 beacon broadcasts, it would be 6 seconds. How about 6?